src/DataProtection/DataProtection/ref/Microsoft.AspNetCore.DataProtection.csproj

```diff
@@ -1,7 +1,7 @@
 <!-- This file is automatically generated. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
Question: is it preferred to duplicate all the shared references for netstandard2.0 vs 3.0 instead of having a reference item group and just having the compile include in the conditioned item groups?

Reply:
It just makes it harder to see if there are any actual difference in refs this way, instead of it being obvious when there is a ref that is specific to only one targetframework

Reply:
We don't hand craft the ref assemblies and this just ends up being what the generator outputs. The src csproj are easier to read and assuming the generator is correct, you shouldn't need to review this file.